sdk/communication/azure-communication-identity/src/main/java/com/azure/communication/identity/CommunicationIdentityClientUtils.java

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.identity;
+
+import com.azure.communication.identity.implementation.models.CommunicationIdentityAccessTokenRequest;
+import com.azure.communication.identity.implementation.models.CommunicationIdentityCreateRequest;
+import com.azure.communication.identity.models.CommunicationTokenScope;
+import com.azure.core.util.logging.ClientLogger;
+
+import java.time.Clock;
+import java.time.Duration;
+import java.time.OffsetDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+final class CommunicationIdentityClientUtils {
+
+    static final String TOKEN_EXPIRATION_OVERFLOW_MESSAGE = "The tokenExpiresIn argument is out of permitted bounds [1,24] hours. Please refer to the documentation and set the value accordingly.";
+    static final double TOKEN_EXPIRATION_ALLOWED_DEVIATION = 0.05;
```

Comment:
it should be in the test package(the best place could be TokenCustomExpirationTimeHelper), in this class we are putting only methods/properties for the client classes. Same for the lines starting from 61 till 110 excluding `getTokenExpirationInMinutes`